### DISCUSSION TITLE
mPDF and WCAG 2.1 /Accessibility

### DISCUSSION THREAD

**Discussion by jerrytom1986** [mpdf/mpdf#2017] [General]
Hi,

I just joined the organization that uses mPDF as the main library, and I have a question related to WCAG 2.1. Currently, all PDFs do not have proper WCAG semantics. When I open them in Adobe Reader, there are no tags at all. Is it possible to change this? Is this a matter of using the proper parameters to generate a PDF?

**Comment by finwe** [comment]
Sure, feel free to propose a PR to add required metadata.

Or, you can sponsor someone to do it.

Or, you can hope someone will notice this and volunteer.
